setup.py

```diff
@@ -371,6 +371,7 @@ def find_packages(lib_directory):
         ###############
         'expander': [],
         'filter': [],
+        'lcov': [],
```

Comment:
I'm not sure about this because there is another lcov option in this same dictionary, should we change the name of the aggregator?